conf/evolutions/default/97.sql

```diff
@@ -0,0 +1,113 @@
+# --- !Ups
+
+UPDATE logged_actions SET action_context_id = -1 where action_context = 2;
+
+DROP VIEW v_logged_actions;
+
+CREATE VIEW v_logged_actions
+   AS
+SELECT
+   a.id,
+   a.created_at,
+   a.user_id,
+   a.address,
+   a.action,
+   a.action_context,
+   a.action_context_id,
+   a.new_state,
+   a.old_state,
+   u.id               as u_id,
+   u.name             as u_name,
+   p.id               as p_id,
+   p.plugin_id        as p_plugin_id,
+   p.slug             as p_slug,
+   p.owner_name       as p_owner_name,
+   pv.id              as pv_id,
+   pv.version_string  as pv_version_string,
+   pp.id              as pp_id,
+   pp.name            as pp_name,
+   pp.slug            as pp_slug,
+   CASE
+       WHEN (a.action_context = 0) THEN a.action_context_id         -- Project
+       WHEN (a.action_context = 1) THEN coalesce(pv.project_id, -1) -- Version
+       WHEN (a.action_context = 2) THEN a.action_context_id         -- ProjectPage
```

Comment:
I couldn't help but notice your first line made anything with `action_context=2` be given an `action_context_id=-1`, so this is always going to return a constant, indeed, the same constant in the `ELSE` block. Perhaps comment out/remove this line? It's a part of the query that you don't need to run.